doc/source/doc-style/docstrings.rst

```diff
@@ -91,11 +99,13 @@ add a docstring for the setter. A setter simply exposes both the GET and SET met
 just the GET method. Examples should be included to demonstrate usage.
 
 Parameters
-~~~~~~~~~~
+----------
 Both classes and methods have parameters in their function signatures. All parameters in a function
```

Comment:
The only difference between a function and a class method is that the second one is a function linked to a class.

I agree the previous line is confusing. Instead, it could read like this:

```suggestion
Functions and class methods may have parameters in their signatures. All these
parameters should be documented in the ``Parameters`` section.
```